leanplum_api.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 1.9.3'
 
-  gem.add_dependency 'activesupport', '~> 3.0'
+  gem.add_dependency 'activesupport', '>= 3.0'
```

Comment:
thanks for the PR, but right now this will auto bundle with rails 5, which requires ruby 2.2.2, and so doesn't work for a lot of people (like me).  maybe let's add `< 5`?

Also are you sure specs will pass?  I'm having issues trying to get them to pass; some issue with `filter_sensitive_data` from the VCR gem aren't working correctly.